### ISSUE TITLE
How to train a better model?

### ISSUE THREAD

**Issue by ghost** [ermongroup/ddim#4]
Hi, I use this code to train a new model, but I find that I cannot get similar FID results.
You successfully provide a new pretrain model on CelebA, so can you give me some advice? Thanks a lot!

**Comment by jiamings** [comment]
I think one of the tricks is to simply train it for longer without it crashing; if it happens, consider reducing the learning rate. The train loss might look the same, but with models that are trained longer the sample quality should get better. Also, don't forget to use EMA in the model.
